commandlineparser/src/Parser.cpp

```diff
@@ -72,7 +72,7 @@ std::vector<std::string> cmdParser::Parser::help_qualifier_keys_finder()
 	return help_keys;
 }
 
-void cmdParser::Parser::default_help(std::vector<std::string> & keys)const
+void cmdParser::Parser::default_help(std::vector<std::string>& keys)const
```

Comment:
make this function print_help(vector, std::function<void(std::shared_ptr<Options>)> print)